pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>diskcheck</artifactId>
-  <version>0.27</version>
+  <version>0.29</version>
```

Comment:
This should be a SNAPSHOT version. You should be releasing with `mvn release:prepare release:perform`.